### PR TITLE
EAP1300 fixes

### PR DESCRIPTION
**Pull request by ryancdotorg**
This re-adds the EnGenius EAP1300 to the firmware update script, and adds the EAP1300EXT as an alt model of the EAP1300.

Thanks to @cminyard for finishing the mac address fixes and DSA conversion.

### CODE REVIEW

**Comment by ryancdotorg** [comment]
These commits should also be pulled in against 23.05 as well, see also #15402

File: target/linux/ipq40xx/base-files/lib/upgrade/platform.sh [hunk 1]
```diff
@@ -115,6 +115,7 @@ platform_do_upgrade() {
 	cilab,meshpoint-one |\
 	edgecore,ecw5211 |\
 	edgecore,oap100 |\
+	engenius,eap1300 |\
```
Comment (tmn505) [inline target/linux/ipq40xx/base-files/lib/upgrade/platform.sh:118]: This looks wrong. Device has NOR flash yet You try to use NAND upgrade procedure. Does the upgrade not work currently? Later in that file eap1300 is handled by asterisk which should suffice.

**Comment by cminyard** [comment]
On Mon, May 06, 2024 at 12:19:40PM -0700, tmn505 wrote:
> @tmn505 commented on this pull request.
> 
> 
> 
> > @@ -115,6 +115,7 @@ platform_do_upgrade() {
>  	cilab,meshpoint-one |\
>  	edgecore,ecw5211 |\
>  	edgecore,oap100 |\
> +	engenius,eap1300 |\
> 
> This looks wrong. Device has NOR flash yet You try to use NAND upgrade procedure. Does the upgrade not work currently? Later in that file eap1300 is handled by asterisk which should suffice. 

This was where it originally was, I think.  I have done about 20
upgrades with this without issue.

**Comment by ryancdotorg** [comment]
@cminyard @tmn505 I had added the device to platform.sh in my previous PR, but I've checked the branches and it was not there originally.

https://github.com/openwrt/openwrt/blob/openwrt-21.02/target/linux/ipq40xx/base-files/lib/upgrade/platform.sh

https://github.com/openwrt/openwrt/blob/openwrt-22.03/target/linux/ipq40xx/base-files/lib/upgrade/platform.sh

I do not remember why I added it before.

**Comment by ryancdotorg** [comment]
I'm closing this, I will submit a PR that just adds the EAP1300EXT.

**Pull request closed without merging**